packages/nitro-protocol/deployment/deploy-test.js

```diff
@@ -0,0 +1,74 @@
+const ethers = require('ethers');
+const etherlime = require('etherlime-lib');
+const path = require('path');
+const writeJsonFile = require('write-json-file');
+const loadJsonFile = require('load-json-file');
+
+const testForceMoveArtifact = require('../build/contracts/TESTForceMove.json');
+const testNitroAdjudicatorArtifact = require('../build/contracts/TESTNitroAdjudicator.json');
+const testAssetHolderArtifact = require('../build/contracts/TESTAssetHolder.json');
+const trivialAppArtifact = require('../build/contracts/TrivialApp.json');
+const countingAppArtifact = require('../build/contracts/CountingApp.json');
+const singleAssetPaymentsArtifact = require('../build/contracts/SingleAssetPayments.json');
+
+const erc20AssetHolderArtifact = require('../build/contracts/ERC20AssetHolder.json');
+const ethAssetHolderArtifact = require('../build/contracts/ETHAssetHolder.json');
+const tokenArtifact = require('../build/contracts/Token.json');
+
+let contractsToAddresses = {};
+
+async function getNetworkMap() {
+  try {
+    return await loadJsonFile(path.join(__dirname, 'network-map.json'));
+  } catch (err) {
+    if (!!err.message.match('ENOENT: no such file or directory')) {
```

Comment:
What's the reason for the `!!` here?

Reply:
`!!anyExpression` is always a boolean value. So, if `anyExpression` is truthy, then `!!anyExpression == true`